src/pages/dashboard/giveaway/index.tsx

```diff
@@ -477,6 +477,23 @@ const GiveawayPage: React.FC = () => {
     setActiveTab("giveaway");
   }, []);
 
+  // Dynamically load Kingsumo embed script for giveaway page
+useEffect(() => {
+  // Create a new script element
+  const script = document.createElement("script");
+  script.src = "https://kingsumo.com/js/embed.js"; // Script URL provided by Kingsumo
```

Comment:
<div id="kingsumo-embed" data-url="https://kingsumo.com/g/1x9doy1/linkedin-giveaway-challenge"></div>
<script src="https://kingsumo.com/js/embed.js"></script>

this is the correct script